README.md

```diff
@@ -15,30 +24,19 @@ Without further ado, here is the Bonusly Employee Un-Handbook.
- * The era of rote and repetitive work is at an end. The jobs of today (& tomorrow) require:
-   * Collaboration, to leverage unique perspectives and expertise across domains.
-   * Creativity, to find novel solutions to difficult problems.
-   * Responsiveness, to adapt to rapidly changing landscape of technologies, customers, and competitors.
- * The way we work must change radically. Current "best practices" are designed for jobs that no longer exist.
+ * Success in business is determined by **people** **working on teams**.
 
 #### This belief inspires us to...
- * Maximize the conditions for collaboration, creativity, and responsiveness:
+ * Maximize the conditions for people and teams to sustainably do their best work over the long run:
    * Purpose - the sense that your job is important and you are having an impact
-   * Transparency - easy access to information about your team's goals and performance
-   * Feedback - on your performance so that you can learn and improve
-   * Community - the bond you share with colleagues as your strive toward a common goals
-   * Empowerment - the confidence and autonomy to experiment, learn, and then take action
+   * Progress - the opportunity to develop your skills and grow as a person, as well as test yourself against new and interesting challenges
+   * Belonging - the feeling that you matter and are valued, combined with the bond you forge with colleagues as your strive toward a common goals
 
 #### We work toward a future in which...
- * Autonomy & ownership is maximized.
- * Information is explicit and shared, never hoarded.
- * Employees are encouraged to experiment, learn, and act.
+ * Every organization achieves success by developing purposeful, inclusive, and high-achieving teams.
 
 ## Values
-These [values are what we value](https://www.slideshare.net/reed2001/culture-1798664/4-Seven_Aspects_of_our_Culture) at Bonusly. They aren’t just nice-sounding words on a poster somewhere, but a succinct description of our culture and what it’s like to work here. We use these values to guide our day-to-day activities and decision-making, from how we respond to customer emails to the roles we hire for to the features we build.
-
-The initial list of these values were created at our first company retreat, when Bonusly was a team of seven people. By Q3 2018 we had grown to 22 people, and decided it was time for an update. We spent a half-day soliciting input from the entire team, which the leadership team then synthesized into an updated set of values that was shared with the whole company in November 2018.
+These values are what we value at Bonusly. They aren’t just nice-sounding words on a poster somewhere, but a succinct description of our culture and what it’s like to work here. We use these values to guide our day-to-day activities and decision-making, from how we respond to customer emails to the roles we hire for to the features we build.
```

Comment:
Something about having values, values, value all within a few words is hurting my brain a bit, and not sure if we need them given the subheading of Values right there.

```suggestion
They aren't just nice-sounding words on a poster somewhere, but a succinct description of our culture and what it’s like to work here. We use these values to guide our day-to-day activities and decision-making, from how we respond to customer emails to the roles we hire for to the features we build.
```